Classes/Controller/BackendServiceController.php

```diff
@@ -75,9 +80,64 @@ public function initializeUpdateAction(): void
      *
      * @return void
      */
-    public function updateAction(array $resultDtos): void
+    public function updateAction(array $resultDtos): string
     {
         $this->assetService->updateMultipleAssets($resultDtos);
-        $this->view->assign('value', $resultDtos);
+        return json_encode(array_map(fn (AssetModuleResultDto $item) => $item->jsonSerialize(), $resultDtos));
+    }
+
+    public function initializeGetFocusKeywordNodesAction(): void
```

Comment:
protected instead of public?

Reply:
Die `initializeFooAction()` Methoden sind immer public.

Reply:
Ah, sorry is eine Action. Ja klar, muss public sein